### PR TITLE
Hit the API to get unique game id, get scores list and post user scores

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### Following changes have been implemented to this PR:

- Read the Leaderboard API documentation to use this API.
- Created a new game with the name "Nabeel's Game" by using the API.
- Saved the ID of the game that was returned by API.
- Implemented the "Refresh" button that gets all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
- Implemented the form "Submit" button that saves a score for a game created (sending data to the API).
- Used arrow functions instead of the function keyword.
- Used async and await JavaScript features to consume the API.
- Corrected Linter errors
- Update Readme file

### CODE REVIEW

File: src/modules/Game.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(header),
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      const resultString = data.result;
+      const code = resultString.split(' ')[3];
+      // console.log('I am code', code);
```
Comment (youmari) [inline src/modules/Game.js]: - [x] it would be better to remove the commented code in order to keep the file and clean

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Metaverse-Nabeel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Game.js (hunk 1 above)
Comment (Metaverse-Nabeel) [reply src/modules/Game.js]: @youmari 
Thank you for your kind feedback. I shall make the required changes and ask again for a review.

**Review by Shisui6 — APPROVED** [review]
Hi @Metaverse-Nabeel, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Fetches and posts data from the API ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**